ibis/sql/sqlite/client.py

```diff
@@ -45,6 +46,7 @@ def __init__(self, path=None, create=False):
         self.database_name = 'default'
 
         self.con = sa.create_engine('sqlite://')
+        self.inspector = Inspector.from_engine(self.con)
```

Comment:
I've changed list_tables in AlchemyClient to use an inspector.  In general, we should probably use inspector where we can since it saves us from having to write dialect specific code.